ovos_persona_server/embeddings.py

```
@@ -0,0 +1,153 @@
+    elif isinstance(request_body.input, list):
+        if all(isinstance(item, str) for item in request_body.input):
+            texts_to_embed = request_body.input
+        else:
+            # This case should ideally be caught by Pydantic validation, but as a fallback
+            raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST,
+                                detail="Input must be a string or array of strings.")
+    else:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST,
+                            detail="Input must be a string or array of strings.")
+
+    model_name: str = embedder.config.get("model",
```

Ruff (0.12.2)

Local variable `model_name` is assigned to but never used

Remove assignment to unused variable `model_name`

(F841)